### PR TITLE
Feature/jr2 119 create mapping table for occupation codes

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: src/main/java/ch/admin/seco/service/reference/domain/OccupationMapping.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    @Column(name = "id", columnDefinition = "uuid")
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "UUIDGenerator")
+    @GenericGenerator(name = "UUIDGenerator", strategy = "uuid2")
+    private UUID id;
+
+    @NotNull
+    @Min(10000000)
+    @Max(99999999)
+    @Column(name = "code", nullable = false)
+    private Integer code;
```
Comment (michael-wirth) [inline src/main/java/ch/admin/seco/service/reference/domain/OccupationMapping.java]: convert to `int`

File: src/main/java/ch/admin/seco/service/reference/domain/OccupationMapping.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+    @NotNull
+    @Min(10000000)
+    @Max(99999999)
+    @Column(name = "x28_code", nullable = false, unique = true)
+    private Integer x28Code;
```
Comment (michael-wirth) [inline src/main/java/ch/admin/seco/service/reference/domain/OccupationMapping.java]: convert to `int`

File: src/main/java/ch/admin/seco/service/reference/domain/OccupationMapping.java [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+    @NotNull
+    @Min(10000)
+    @Max(99999)
+    @Column(name = "avam_code", nullable = false)
+    private Integer avamCode;
```
Comment (michael-wirth) [inline src/main/java/ch/admin/seco/service/reference/domain/OccupationMapping.java]: convert to `int`

File: src/main/java/ch/admin/seco/service/reference/repository/OccupationMappingRepository.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+
+/**
+ * Spring Data JPA repository for the OccupationMapping entity.
+ */
+@SuppressWarnings("unused")
+@Repository
+public interface OccupationMappingRepository extends JpaRepository<OccupationMapping, UUID> {
+
+    // TODO: this will return a list as we have duplicates
```
Comment (varusakov) [inline src/main/java/ch/admin/seco/service/reference/repository/OccupationMappingRepository.java]: @michael-wirth could I remove TODOs?

Comment (michael-wirth) [inline src/main/java/ch/admin/seco/service/reference/repository/OccupationMappingRepository.java]: yes please remove them as well

**Comment by michael-wirth** [comment]
unfortunately the squished commit doesn't follow our naming convention. i'll rename it tomorrow

**Pull request merged**